### PR TITLE
Anjuta (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**
[Anjuta](http://anjuta.org) is the GNOME Integrated Development Environment.
It has a hard dependency on GDL, another Gnome project, that was not yet in Homebrew, so I added a formula for that one as well

### CODE REVIEW

File: Library/Formula/anjuta.rb [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  depends_on "gnome-common" => :build
+  depends_on "bison" => :build
+  depends_on "gtksourceview3"
+  depends_on "libxml2"
+  depends_on "libgda"
+  depends_on "gdl"
+  depends_on "vte3"
+  depends_on "hicolor-icon-theme"
+  depends_on "shared-mime-info"
+  depends_on :python
```
Comment (DomT4) [inline Library/Formula/anjuta.rb]: Needs to be either `depends_on :python if MacOS.version <= :snow_leopard` or `depends_on "python"`. Try for the former, we've been trying to eradicate the latter.

**Comment by tschoonj** [comment]
@DomT4 done!

**Comment by tschoonj** [comment]
@DomT4 can you please restart this build? something went wrong on one of the bots

Thanks!

**Comment by DomT4** [comment]
Done. Mountain Lion is being grumpy this week.

File: Library/Formula/anjuta.rb [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  depends_on "vte3"
+  depends_on "hicolor-icon-theme"
+  depends_on "gnome-icon-theme"
+  depends_on "shared-mime-info"
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "vala" => :recommended
+  depends_on "autogen" => :recommended
+
+  # patch submitted upstream at https://bugzilla.gnome.org/show_bug.cgi?id=752763
+  patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/anjuta.rb]: Probably want to wait for this to get merged in some form before we merge this.

**Comment by tschoonj** [comment]
@mikemcquaid Two commits have been made upstream that fix this issue based on my patch (https://github.com/GNOME/anjuta/commit/fb8f9925bc64cba682ad697021d78c78c9ca73d5 and https://github.com/GNOME/anjuta/commit/25e5c3f3f0ec2b4f6770897f07a586bf427f74ce)

I cannot include these commits as patches though due to changes in the anjuta development branch since 3.16.0

Could you pull this PR in as it is now?

Thanks

**Comment by mikemcquaid** [comment]
Thanks @tschoonj!

**Pull request closed without merging**